documentation/docs/adapters/MythicMobsAdapter/entries/fact/mob_exist_fact.mdx

```diff
@@ -0,0 +1,19 @@
+import {FactsField, ReadonlyFactInfo} from "@site/src/components/EntryField";
+
+# Mob Exist Fact
+
+A [fact](/docs/facts) that represents whether a MythicMobs is in the world.
+
+<ReadonlyFactInfo />
+
+## How could this be used?
+
+Change dialogue when a boss exist.
```

Comment:
Try to make this more descriptive.

The `How can this be used?` section is meant to give new users ideas of how to use an entry. Give ideas of scenarios where this entry could be useful.